generators/angular/templates/src/main/webapp/app/entities/_entityFolder_/list/_entityFile_.component.html.ejs

```diff
@@ -88,11 +90,14 @@
 <%_ for (const relationship of relationships.filter(rel => !rel.otherEntityIsEmbedded)) { _%>
   <%_ if (relationship.persistableRelationship && (!relationship.collection || paginationNo)) {
     const fieldName = "." + relationship.otherEntityField;
+    const fieldIsSortable = !relationship.relatedField.transient;
```

Comment:
I prefer to add as a field property in bootstrap-application generator and maybe rename to `relationshipSupportsSortBy`.